app/src/main/java/com/todoist_android/data/network/APIAuthentication.kt

```diff
@@ -21,4 +26,15 @@ interface APIAuthentication {
         @Field("email") email: String,
         @Field("password") password: String
     ) : SignupResponse
+
+
+    @POST("/tasks")
+    suspend fun addTasks(@Body tasksRequest: AddTaskRequest): AddTasksResponse
+
+    @POST("/tasks")
+    suspend fun editTasks(@Body editTasksRequest: TodoModel): TodoModel
+
+    @DELETE("/tasks")
+    suspend fun deleteTasks(@Body deleteTaskRequest:DeleteTaskRequest): DeleteTaskResponse
```

Comment:
this can move to `src/main/java/com/todoist_android/data/network/TaskApi.kt`

Reply:
yes